### PR TITLE
Projects search

### PR DESCRIPTION
**Pull request by danielyuenhx**
Added search for projects page in reference to #6. Also minor fixes for Footer UI

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -56,6 +57,10 @@ export default function Footer() {
     },
   ];
 
+  const hashtags = [
```
Comment (gregv) [inline src/components/Footer/Footer.js:60]: Definitely makes things cleaner if we want to add/remove more

File: src/components/NonProfitList/NonProfitList.js [hunk 2]
```diff
@@ -39,47 +35,9 @@ function NonProfitList() {
...
-            paddingLeft: `calc(1em + ${theme.spacing(4)})`,
-            transition: theme.transitions.create('width'),
-            width: '100%',
-            [theme.breakpoints.up('md')]: {
-                width: '20ch',
-            },
-        },
-    }));
-    */
+    const onChangeSearchHandler = (event) => {
```
Comment (gregv) [inline src/components/NonProfitList/NonProfitList.js:38]: Props for seeing the stubbed out/commented out/dead code and replacing it!

File: src/components/NonProfitList/NonProfitList.js [hunk 3]
```diff
@@ -135,66 +93,84 @@ function NonProfitList() {
...
             Loading... <Puff stroke="#0000FF" /> <Puff stroke="#0000FF" />
             </p>
         );
-    }
+      }
+      
+      if (searchString) {
+        result = result.filter(
+          nonprofit =>
+            nonprofit.name.toLowerCase().includes(searchString.toLowerCase()) ||
```
Comment (gregv) [inline src/components/NonProfitList/NonProfitList.js:109]: Nice! I like that you also added description to this search criteria

**Review by gregv — APPROVED** [review]
The only _minor_ thing to think about that we'd also see within industry are:
1/ Test plans (things you did manually or automated to verify this works)
2/ Screenshots that show before and after

Very awesome addition!

**Comment by danielyuenhx** [comment]
Thanks for the advice! Will try to continue work on #6 when I am free.

**Pull request merged**